pattern/dedicated-community-leader

nyeates opened the pull request:
When starting an InnerSource initiative it is crucial to nominate the right people to lead the communities. Selecting the wrong persons and/or not providing enough capacity for them risks wasted effort and ultimately the failure of the InnerSource initiative. The person needs both communication skills as well as technical.

Reviewers not yet on github:
Nigel Green
Diogo Fregonese

Commit:
Create from wiki

Note that this file exists already in master, but that we need to get it into proper PR review process.

gruetter commented on dedicated-community-leader.md:
```diff
@@ -0,0 +1,80 @@
+
+## Context
+
+- Company A is a large and old company. It has no prior experience in Open Source or other, community based working models. The company culture is best characterized as a classical top down management style - it is generally at odds with community culture.
```
Bob: "The company is large and old".

gruetter commented on dedicated-community-leader.md:
```diff
@@ -0,0 +1,80 @@
+
+## Problem
+
+When starting an InnerSource initiative it is crucial to nominate the right people to lead the communities. Selecting the wrong persons and/or not providing enough capacity for them risks wasted effort and ultimately the failure of the InnerSource initiative.
```
Bob: boldface first paragraph of problem statement

gruetter replied:
Erin: Try to condense into one sentence.

gruetter replied:
Erin: "... the failure of the ..." -> "... the failure of the **new** ..."

gruetter replied:
@NewMexicoKid : "How do you ensure that the new InnerSource initiative has the right leadership for the communities?"

gruetter commented on dedicated-community-leader.md:
```diff
@@ -0,0 +1,80 @@
+
+When starting an InnerSource initiative it is crucial to nominate the right people to lead the communities. Selecting the wrong persons and/or not providing enough capacity for them risks wasted effort and ultimately the failure of the InnerSource initiative.
+
+Consider the following story. Company A wants to initiate an InnerSource initiative in order to foster collaboration across organizational boundaries. They have decided to start with an experimental phase with limited scope. Management has selected a suitable pilot topic for the first InnerSource community and expects contributions from many business units across the organization. Company A has n ...
```
Bob: Italicize the story. Alernatively, move _story_ before _Problem_

Commit:
fixed problem statement and story

Added story title, shortened problem statement, while also keeping kicker

Commit:
changed title to community manager

The industry term is pretty standard on "Community Manager"; added link to popular book

NewMexicoKid commented on dedicated-community-leader.md:
```diff
@@ -0,0 +1,81 @@
+## Title
+
+_Dedicated Community Manager_
```
If you change the title to Community Manger, you should highlight the importance of leadership to this role (somewhere in the pattern; perhaps context? Problem?)

gruetter replied:
done with 13cefcb

gruetter replied:
@HaRo87 : The community brought up the point that the term _Community Manager_ is commonly used in literature (e. g. by Jono Bacon in his book [The Art of Community](http://www.artofcommunityonline.org)... even though he calls his summit the _Community Leadership Summit_ ;). This is why we changed the title. I did add statements on the importance of leadership in the _Forces_ and _Solution_ sections.

NewMexicoKid commented on dedicated-community-leader.md:
```diff
@@ -0,0 +1,81 @@
+
+## Story
+
+Consider the following story. A company wants to initiate an InnerSource initiative in order to foster collaboration across organizational boundaries. They have decided to start with an experimental phase with limited scope. Management has selected a suitable pilot topic for the first InnerSource community and expects contributions from many business units across the organization. The company has ...
```
initiate -> start or launch

NewMexicoKid commented on dedicated-community-leader.md:
```diff
@@ -0,0 +1,81 @@
+
+- The company is a large and old company. It has no prior experience in Open Source or other, community based working models. The company culture is best characterized as a classical top-down management style - it is generally at odds with community culture.
+- While there are supporters and a sponsor in top level management, middle management in the company is not yet sold on InnerSource.
+- Management has provided a limited budget to fund a part time community leader only.
```
Management was not convinced to provide more than a limited budget to fund a part time community leader only.

ErinMB commented on dedicated-community-leader.md:
```diff
@@ -0,0 +1,81 @@
+
+The value contribution of InnerSource projects will not be obvious for many managers which are steeped in traditional project management methods. Those managers are less likely to assign one of their top people, who are usually in high demand by non InnerSource-projects, to an InnerSource project for a significant percentage of their work time.
+
+Community work - especially communication - make up for a significant percentage of a community leaders daily work. At the same time, he will likely also have to spearhead the initial development, too. In the face of limited capacity, inexperienced leaders will tend to focus on development and neglect communication. The barrier for potential contributors to make their first contribution and to co ...
```
Communication takes up a significant percentage of a community manager’s daily work. At the same time, he or she will likely also have to spearhead the initial development.

NewMexicoKid replied:
Be sensitive to gender of pronouns throughout this and other patterns (he or she or they).

NewMexicoKid commented on dedicated-community-leader.md:
```diff
@@ -0,0 +1,81 @@
+Community work - especially communication - make up for a significant percentage of a community leaders daily work. At the same time, he will likely also have to spearhead the initial development, too. In the face of limited capacity, inexperienced leaders will tend to focus on development and neglect communication. The barrier for potential contributors to make their first contribution and to co ...
+
+If a community can not grow fast enough and pick up enough speed, chances are they won't be able to convincingly demonstrate the potential of InnerSource.
+
```
Add a force about the potential lack of leadership in a community manager who is focused only on managing a project (focused on the structure and resources and less on leading the people, being a role model, etc).

NewMexicoKid commented on dedicated-community-leader.md:
```diff
@@ -0,0 +1,81 @@
+- has the required soft-skills to act as a natural leader,
+- is an excellent networker and who
+- inspires community members.
+
```
leads by example (to be credible to the community).

NewMexicoKid replied:
Management should be informed of the need to be sensitive to the views of the open community before they engender a change in community manager or before choosing a community manager.

nyeates commented on dedicated-community-leader.md:
```diff
@@ -0,0 +1,81 @@
+- inspires community members.
+
+Empower the community leader to dedicate 100 % of his time to community work including communication and development. 
+
```
This person should both be a leader and a manager of the efforts.

NewMexicoKid replied:
Point to or reference other articles or books on this topic.

ErinMB commented on dedicated-community-leader.md:
```diff
@@ -0,0 +1,81 @@
+- has the required soft-skills to act as a natural leader,
+- is an excellent networker and who
+- inspires community members.
+
```
Can communicate effectively to both executive management and Developers

NewMexicoKid commented on dedicated-community-leader.md:
```diff
@@ -0,0 +1,81 @@
+- Tim Yao
+- Padma Sudarsan
+- Nigel Green
+
```
Add Nick, Erin and Daniel

NewMexicoKid commented on dedicated-community-leader.md:
```diff
@@ -0,0 +1,81 @@
+
+## Changelog
+
+- **2016-11-06** - first review
```
2017-04-06 - second review

nyeates commented on dedicated-community-leader.md:
```diff
@@ -0,0 +1,81 @@
+**Review comments**
+- (**Open**: I have added something to the problem statement instead) now added that to the maybe reference pattern tbd (start small, experiment then scale up as it has proven successful)
+
+## Solution
```
Reference chapter in "Art of Community" around choosing a good community manager.
Daniel says to Include "Producing Open Source" book reference

gruetter commented on dedicated-community-leader.md:
```diff
@@ -0,0 +1,81 @@
+
+_Dedicated Community Manager_
+
+alternative:_Dedicated Community Leader_
```
I will remove this alternative title. I'll highlight the difference between management and leadership below.

gruetter replied:
done with 13cefcb

Commit:
factored in pull request feedback. Thanks Tim, Padma, Erin, Nick and Daniel!
